extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -502,10 +515,34 @@ export class AmpStoryEmbeddedComponent {
     this.focusedStateOverlay_.addEventListener('click', event =>
       this.onOutsideTooltipClick_(event)
     );
+    this.tooltip_.addEventListener(
+      'click',
+      () => this.tooltipAnalytics_(),
+      true /** capture */
+    );
 
     return this.shadowRoot_;
   }
 
+  /**
+   * Configures analytics variables and fires analytic event.
+   * @private
+   */
+  tooltipAnalytics_() {
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.TOOLTIP_TARGET,
+      this.triggeringTarget_.tagName.toLowerCase()
```

Comment:
I guess in general some variables aren't useful by themselves, only when joined 
together. e.g. in the bookend "card-type" isn't very useful by itself, but having the `href` along in the request helps identifying what component was clicked.

If i'm a publisher and want to know what element triggered the tooltip I think having the tagname and the "id" (see other comment) would be enough to identify it.

Do you have any other ideas of identifiers we could use?

Reply:
I'd be curious what amp-analytics normally does for this.  For example, in an AMP website, if I set up a click listener on all `<a>` tags, are there any variables included that help me tell all the `<a>` tags on my page apart?

/cc @zhouyx

Reply:
You can do that via the `data-vars-` attribute https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md#variables-as-data-attribute

There're also requests to include `<a>` tag url along with the request if asked. Which I think it's reasonable, but we haven't got chance to address. Please let me know what best address your use case here?

Reply:
Thanks for the insights!  The requests you've received are just too include the tag name?  Or some other string representation of the element?

Reply:
We received the request to include the href of an anchor tag, and src of an image by default, so that one doesn't need to copy the url to a data-vars attribute. We haven't received request to include tag name though.

Reply:
Should we just use the href/data-tweetid/... then? WDYT @newmuis?

Reply:
Seems to me like we might get more bang for the buck by supporting the "variables as data attributes" approach above, since if they wanted to, a publisher could always include that information in their data vars.

Conversely, if we only populate e.g. the href, they couldn't include custom information.  For example, I remember one publisher who had pages that could have two types of links.  Without the ability to include these custom vars, you would have to know all of the URLs and have some processing pipeline to determine which type of link you have.  (In that publisher's case, they made them CTA links so that they could avoid the shadow boundary altogether, encode the "type" into a CSS class, and filter the analytics trigger by selector)

Reply:
I agree with Jon here. `data-vars-` approach is more generic so that you don't need to handle info from different components differently. 
It also depends on what are the event information you included. Are they more like custom information from publisher, or defined information that you want to define the name and the meaning. 

From the example, I read
```
         <amp-twitter width="375" data-tooltip-text="my twitter embed"
            height="472"
            layout="responsive"
            data-tweetid="1166723359696130049"
            data-vars-tooltip-click-id="tweet1"
            data-vars-tooltip-href="twitter.com"
            interactive>
          </amp-twitter>
```
It is `tooltip-click-id` and `tooltip-href` just random value and name, or do you always want to pass something like id and href.

Reply:
Yep, those are just random values I thought of for the variable names. It would support any value following the `data-vars-*` rule.